.travis.yml

```diff
@@ -109,4 +109,4 @@ after_success:
             lcov --remove coverage.info "/usr/*" "*Xcode.app*" "*/tests/*" --output-file coverage.info
             lcov --list coverage.info
             bash <(curl -s https://codecov.io/bash) -f coverage.info || echo "Upload to codecov.io FAILED"
-        fi
+        fi
```

Comment:
Unneeded change. Please add new line at the end of file and configure your IDE to always add a new line at the end of file. In VSCode it can be done by settings `"editor.formatOnSave": true` (or similar property).